util/src/main/java/tech/pegasys/artemis/util/time/DateEventsChannel.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.util.time;
+
+import java.util.Date;
+
+public interface DateEventsChannel {
```

Comment:
Maybe `TimeEventsChannel` or `TimeTickChannel`?

Reply:
TimeTick sounds good. I don't want to say TimeEvents because SlotEvents is also a time event yet it is on a different channel.

Reply:
Went with TimeTickChannel because SlotEvents are also time events, but they're on a different channel.